localstack/services/opensearch/resource_providers/aws_opensearchservice_domain.py

```diff
@@ -223,6 +224,19 @@ def create(
                 cluster_config.setdefault("DedicatedMasterType", "m3.medium.search")
                 cluster_config.setdefault("WarmType", "ultrawarm1.medium.search")
 
+                for key in ["DedicatedMasterCount", "InstanceCount", "WarmCount"]:
+                    if key in cluster_config and isinstance(cluster_config[key], str):
+                        cluster_config[key] = int(cluster_config[key])
+
+            if properties.get("AccessPolicies") and isinstance(properties["AccessPolicies"], dict):
```

Comment:
can this actually already be a string? don't think we have a test for this right?